src/JoomlaBrowser.php

```diff
@@ -911,7 +911,11 @@ public function createMenuItem($menuTitle, $menuCategory, $menuItem, $menu = 'Ma
 		$I->switchToIFrame();
 		$I->debug('I leave time to the iframe to close');
 		$I->wait(2);
-		$I->selectOptionInChosen('Language', $language);
+
+		// @Todo We should use page objects for the selectors
+		$I->scrollTo(['css' => '#jform_language'], 20, 100);
```

Comment:
Just a side note: The sticky nav bar is not optimal for this kind of testing. I added a step in my test cases which disables this feature in the template options.